src/app/tourist/book/page.tsx

```
@@ -40,17 +37,30 @@ function BookDestinationForm() {
+    specialRequests: '',
+    acknowledged: false
   });
-  const [dateError, setDateError] = useState<string | null>(null);
+  const [errors, setErrors] = useState<Record<string, string>>({});
 
-  const loadDestination = useCallback(async () => {
+  useEffect(() => {
+    if (destinationId) {
+      loadDestination();
+    }
+  }, [destinationId]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)